### DISCUSSION TITLE
Is there a reason why ui/dist is tracked and not added to gitignore?

### DISCUSSION THREAD

**Discussion by alfredjophy** [pocketbase/pocketbase#3830] [Q&A]
I was just working on adding some features for a personal use case and noticed that the ui build isn't ignored.
I am just curious if there is a reason.

Thanks

**Comment by ganigeorgiev** [comment]
It is part of the repo so that it can be embedded and ready to use out of the box (see [embed.go](https://github.com/pocketbase/pocketbase/blob/master/ui/embed.go)) without requiring users to have node or to build the static frontend assets on their own when [extending PocketBase with Go](https://pocketbase.io/docs/go-overview/).
